m4_sessions/1705-inning/group_five.md

```diff
@@ -16,3 +14,45 @@ To make students realize the psychological and sociological impacts of social me
 * Jon W.
 * Mickey Fen
 * Spencer Carter
+
+## SESSION OUTCOMES
+
+* Realization of implications of social media on yourself and others
+
+* Making people think about their social presence (+/-)
```

Comment:
What do you want them to consider about their social presence?  The pros/cons?  The impact it has?  How they are using it?